code/modules/clothing/head/misc_special.dm

```diff
@@ -189,6 +189,7 @@
 	flags_inv = HIDEHAIR
 	var/hair_style = "Very Long Hair"
 	var/hair_color = "#000"
+	var/fixedcolor = TRUE //can color be changed manually?

```

Comment:
the var is called "fixedcolor" but setting it to TRUE means you can change the color, aka it's not fixed.

Reply:
I my mind it was "fixed color" as in "it doesn't change on its own" but I get the confusion, I'll change to adjustablecolor